### ISSUE TITLE
Fetching from alpine-linux repository is not allowed by self-hosted runner because of using http protocol by default 

### ISSUE THREAD

**Issue by tlytvynDoosan** [simple-elf/allure-report-action#53]
I'm using a self-hosted runner and by default port 80 is blocked by company policy. Using the action I get error
```
 #6 0.465 fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/main/x86_64/APKINDEX.tar.gz
  #6 134.2 ERROR: http://dl-cdn.alpinelinux.org/alpine/v3.9/main: network error (check Internet connection and firewall)
```

because url uses http protocol by default. Is there a way to force docker to use https protocol? Or update action so it will use https for fetching images from a repo

**Comment by mgrybyk** [comment]
@tlytvynDoosan I'm suggesting two options:
- use https://github.com/mgrybyk-org/allure-report-branch-js-action that doesn't use Docker at all (Java is required to be installed).
- use https://github.com/mgrybyk-org/allure-report-branch-action it doesn't make any http calls

**Comment by tlytvynDoosan** [comment]
@mgrybyk Yeah man thanks a lot for this. I'm already making transition to your actions
